### DISCUSSION TITLE
spack terminated by signal SIGBUS (Misaligned address error)

### DISCUSSION THREAD

**Discussion by jbr** [swc-project/swc#1434] [Q&A]
Hi! Thanks for swc!

I'm looking into using swc in an asset pipeline for a rust web framework, but on my initial exploration I ran into this opaque error on latest macos (intel):

```
$ npx spack
fish: 'npx spack' terminated by signal SIGBUS (Misaligned address error)
```

spack.config.js:
```js
const { config } = require('@swc/core/spack');

module.exports = config({
  entry: { web: __dirname + '/src/index.tsx' },
  output: { path: __dirname + '/build' },
  module: {},
});
```

.swcrc:
```json
{
  "jsc": {
    "parser": {
      "syntax": "typescript",
      "tsx": true,
      "decorators": false,
      "dynamicImport": false
    },
    "target": "es2016"
  },
  "minify": true
}
```

The straightforward react app bundles just fine with snowpack and also builds with `npx swc src/*.tsx`. I've never encountered this error before — is it an issue with my environment / configuration or a bug in spack?

**Comment by kdy1** [comment]
What are your os and CPU?
